Index: pbbuttonsd-0.8.1a/libpbbipc/pbb_errno.c
===================================================================
--- pbbuttonsd-0.8.1a.orig/libpbbipc/pbb_errno.c	2015-01-05 14:08:11.613978849 +0100
+++ pbbuttonsd-0.8.1a/libpbbipc/pbb_errno.c	2015-01-05 14:08:13.202772958 +0100
@@ -108,9 +108,9 @@
  */
 	if (base->daemon != PBBDS_NONE) {
 		if (type == PBB_ERR && base->daemon != PBBDS_CHILD)
-			fprintf(stderr, inptr);
-		syslog(level, inptr);
+			fputs(inptr, stderr);
+		syslog(level, "%s", inptr);
 	} else
-		fprintf(stderr, inptr);
+		fputs(inptr, stderr);
 }
 
