Description: Handle mac mini power button
Author: Mathieu Malaterre <malat@debian.org>

--- pbbuttonsd-0.8.1a.orig/src/module_pmac.c
+++ pbbuttonsd-0.8.1a/src/module_pmac.c
@@ -552,7 +552,7 @@ pmac_pmu_handler (int fd, gpointer user_
 			/* n = 2 && intr[1] = 0x0c = %01100 power button on mac-mini */
 			/* n = 6 && intr[1] = 0x1c = %11100 if power button is pressed */
 			/* n = 6 && intr[1] = 0x15 = %10101 after cover close */
-			if (n == 6 && ((intr[1] >> 3) & 1) != PBpressed) {
+			if (n == 2 && intr[1] == 0x0c ) {
 				PBpressed = (intr[1] >> 3) & 1;
 				taglist_init (taglist);
 				taglist_add (taglist, TAG_KEYCODE, KEY_POWER);
@@ -563,8 +563,7 @@ pmac_pmu_handler (int fd, gpointer user_
 			break;
 		}
 #if defined(DEBUG) && PMUINTR
-#error bla
-		printf ("\nPMU interrupt: ");
+		printf ("\nPMU interrupt: (%d)\n", n);
 		for (i=0; i < n; i++)
 			printf ("%.2x ", intr[i]);
 		printf ("\n");
